### PR TITLE
32-bit support removal from examples/samples

### PR DESCRIPTION
**Pull request by napetrov**
In this PR we are removing all 32-bit mentions in examples and samples

### CODE REVIEW

File: examples/daal/cpp/daal_win.vcxproj.tpl [hunk 1]
```diff
@@ -73,18 +41,7 @@
...
-    <CharacterSet>MultiByte</CharacterSet>
-    <PlatformToolset>v141</PlatformToolset>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug.static.sequential|Win32'" Label="Configuration">
-    <ConfigurationType>Application</ConfigurationType>
-    <UseDebugLibraries>true</UseDebugLibraries>
-    <CharacterSet>MultiByte</CharacterSet>
-    <PlatformToolset>v141</PlatformToolset>
-  </PropertyGroup>
+
```
Comment (vepifanov) [inline examples/daal/cpp/daal_win.vcxproj.tpl:44]: Blank line.

Comment (napetrov) [reply examples/daal/cpp/daal_win.vcxproj.tpl]: thanks

**Pull request merged**